src/ua/lviv/lpnu/icta/iot/mykola/kovalyk/lab1/Main.java

```diff
@@ -0,0 +1,22 @@
+package ua.lviv.lpnu.icta.iot.mykola.kovalyk.lab1;
+
+import java.util.Arrays;
+
+public class Main {
+
+	public static void main(String[] args) {
+		// TODO Auto-generated method stub
+		
+		var gasMeter1 =  new GasMeter();
+		var gasMeter2 =  new GasMeter(3.f, "VasyaGasMeter", 5.2f);
+		var gasMeter3 =  new GasMeter(3.5f, "PetyaGasMeter", 10.3f);
+		
+		System.out.println(gasMeter1.toString());
```

Comment:
you dont need to call toString()
its called automatically from your overriden method

Reply:
Redundancy removed, thanks.